### ISSUE TITLE
Update v0.9.0-beta.2 fails

### ISSUE THREAD

**Issue by daemonenstall** [kodebach/hacs-idm-heatpump#132]
Hi,

I can't update because the file that is announced via HACS does not exist.

```
Logger: custom_components.hacs
Quelle: custom_components/hacs/base.py:709
Integration: HACS ([Dokumentation](https://hacs.xyz/docs/configuration/start), [Probleme](https://github.com/hacs/integration/issues))
Erstmals aufgetreten: 21:31:20 (1 Vorkommnisse)
Zuletzt protokolliert: 21:31:20
Download failed - Got status code 404 when trying to download https://github.com/kodebach/hacs-idm-heatpump/releases/download/v0.9.0-beta.2/idm_heatpump.zip

Traceback (most recent call last):
  File "/config/custom_components/hacs/base.py", line 709, in async_download_file
    raise HacsException(
custom_components.hacs.exceptions.HacsException: Got status code 404 when trying to download https://github.com/kodebach/hacs-idm-heatpump/releases/download/v0.9.0-beta.2/idm_heatpump.zip
```

**Comment by kodebach** [comment]
Seems there is an issue with the automated release action that builds the ZIP file for HACS. I'll look into it.

**Comment by kodebach** [comment]
I just released `v0.9.0-beta.3`, if the download fails again please reopen
